.github/workflows/e2e.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   e2e:
     name: E2E
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
This doesnt fix the tests though, theres some time out there, could it be a deadlock or something of the sort causing the tests to time out and fail?

It seems to be happening mostly on operator tests

Reply:
I freed more space, hopofully will be more stable now, but last run passed :partying_face: